docs/types/PanelCursorTransitionType.md

```diff
@@ -0,0 +1,9 @@
+---
+id: panelCursorTransitionType
+title: PanelCursorTransitionType
+---
+
+# PanelEdgeConstraintLevel options:
```

Comment:
this should be Cursor not Edge

Reply:
Thank you @kpiascik . I have updated the label !